lib/screens/app/wallet/receive.dart

```diff
@@ -742,7 +750,7 @@ class _ReceiveFormState extends State<ReceiveForm> {
               Row(
                 children: [
                   Text(
-                    price.toString(),
+                    price.toStringAsFixed(4),
```

Comment:
We have formatters for Seeds and also fiat currency, please use those, instead of "fixed 4"


See Class DoubleExtension <-----